packages/design-system/src/Tooltip/Tooltip.types.tsx

```diff
@@ -0,0 +1,30 @@
+import type { TooltipProps as RcTooltipProps } from "rc-tooltip/lib/Tooltip";
+
+type Placement =
+  | "left"
+  | "right"
+  | "top"
+  | "bottom"
+  | "topLeft"
+  | "topRight"
+  | "bottomLeft"
+  | "bottomRight"
+  | "rightTop"
+  | "rightBottom"
+  | "leftTop"
+  | "leftBottom";
+
+// Tooltip props
+export type TooltipProps = {
+  /** Tooltip content to be shown */
+  content: (() => React.ReactNode) | React.ReactNode;
```

Comment:
I don't think we should let anyone pass anything. We should restrict it to the two  types of content shown in design.